src/app/core/translations/pl_PL.ts

```diff
@@ -155,4 +155,6 @@ export default {
   NO_RESULTS_TEXT_START: 'Zmień kryteria wyszukiwania, lub',
   NO_RESULTS_TEXT_LINK: 'przejdź do informatora',
   NO_RESULTS_TEXT_END: 'by wyświetlić inne dostępne formy pomocy dla tej kategorii.',
+  MESSAGE_SENT_ALERT_HEADER: 'Twoja odpowiedź została wysłana do osoby pomagającej',
```

Comment:
Missing dot at the end of the sentence, is it intended?

Reply:
@maciejp-github This is a header so maybe yes?